tests/doc-links.spec.ts

```diff
@@ -0,0 +1,235 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  editInstance,
+  randomInstanceName,
+} from "./helpers/instances";
+import {
+  createProject,
+  deleteProject,
+  randomProjectName,
+} from "./helpers/projects";
+import {
+  validateServerSettingLink,
+  validateDocumentationLink,
+} from "./helpers/doc-links";
+import { visitServerSettings } from "./helpers/server";
+import { activateOverride } from "./helpers/configuration";
+
+test("Ensure the documentation link text and link targets are present: Server Settings > Override", async ({
+  page,
+  lxdVersion,
+}) => {
+  test.skip(
+    lxdVersion === "5.0-edge",
+    "Generated documentation only started shipping with newer versions of LXD",
+  );
+  await visitServerSettings(page);
+
+  await validateServerSettingLink(
+    page,
+    "Separate REST API and clustering networks",
+    "/documentation/howto/cluster_config_networks/#cluster-https-address",
+    "cluster.https_address",
+  );
```

Comment:
I think here was a misunderstanding. I'd suggest splitting the function to `editServerSetting` so the code becomes very descriptive: 
```suggestion
  await editServerSetting(page, "cluster.https_address");
  await validateDocumentationLink(
    page,
    "Separate REST API and clustering networks",
    "/documentation/howto/cluster_config_networks/#cluster-https-address"
  );
```

Reply:
Ah, I understand now, I'll action that.